libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -109,6 +109,21 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("RF_SELECT",   9, AP_ADSB, out_state.cfg.rfSelect, UAVIONIX_ADSB_OUT_RF_SELECT_RX_ENABLED),
 
+    // @Param: SQUAWK
+    // @DisplayName: Squawk code
+    // @Description: VFR squawk (Mode 3/A) code is a pre-programmed default code when the pilot is flying VFR and not in contact with ATC. In the USA, the VFR squawk code is 1200 [0x04B0] and in most parts of Eurpope the VFR squawk code is 7000.
+    // @Unit: decimal
```

Comment:
Isn't that 1200 octal ?
AFAIK decimal is not a valid unit.

Reply:
I believe it's 1200 decimal because it's hex 04B0. It would make sense to be octal.. I went back and forth on it. If it's decimal then 0x3FFF is correct too

Reply:
If 1200 is in octal, which I believe it is, then the hex code is 0x0280 and the 0x0FFF bitmask is correct.